tests/unit/test_search_results.py

```diff
@@ -0,0 +1,75 @@
+"""Module to test the KytosGraph in graph.py."""
+from unittest import TestCase
+
+import networkx as nx
+# Core modules to import
+from kytos.core.interface import Interface
+from kytos.core.link import Link
+from kytos.core.switch import Switch
+
+# module under test
+from graph import KytosGraph
+
+
+class TestSearchResults(TestCase):
+    """Tests for the graph class."""
+
+    def setup(self):
+        """Setup for most tests"""
+        switches, links = self.generate_topology()
+        self.graph = KytosGraph()
+        self.graph.clear()
+        self.graph.update_nodes(switches)
+        self.graph.update_links(links)
+        self.graph.set_path_fun(nx.shortest_simple_paths)
```

Comment:
You are testing using `nx.shortest_simple_paths` as the path function, but in the actual code only `nx.all_shortest_paths` is used. Why are you testing with a different function?

Reply:
`nx.shortest_simple-paths` returns a subset of `nx.all_shortest_paths`, namely the paths that do not have repeating nodes. So it was likely added to save time testing the new graph implementation.

I removed the call to the setter, to use `nx.all_shortest_paths` instead, and did not see a regression in the test results.